### PR TITLE
RevAutomateStudio_2topleveldocs

### PR DESCRIPTION
**Pull request by lorisock**
Rewrote content to better adhere to Style Guide and removed some superfluous sections of both docs (What's New in Automate Studio doc and detailed JavaScript object information in application doc.

### CODE REVIEW

File: src/docs/automate-studio/application/README.md [hunk 1]
```diff
@@ -1,99 +1,63 @@
...
 
 ### Message (msg)
 
 All flows in Automate Studio rely on a **msg** object to pass information from node to node. The **msg** object is created automatically by Automate Studio. It is an all-purpose object that you can modify or just use as needed. **msg** has global scope and visibility, which means that it is visible (accessible and usable) to all nodes and all sub-flows of a flow.
 
 The **msg** object is a JavaScript object (also called JSON object) and contains a **payload** property (or "field") by default. The value of the **payload** property  changes as various nodes of the flow process our data. You can attach a Debug node to any node and check the Debug panel after running the flow to review the **payload** field's value.
 
->**Tip:** To see how the msg object changed from node to node, you can wire Debug nodes to some or all of our flow's nodes. You must then set each Debug node to output "complete msg object" then run the flow to see messages from all Debug nodes in the debug panel in the sidebar.
```
Comment (myoungerman) [inline src/docs/automate-studio/application/README.md:50]: Did you intend to delete this section, or was it an accident? I don't know enough about it to tell if it should stay.

**Review by myoungerman — CHANGES_REQUESTED** [review]
Left 1 comment before I can approve

File: src/docs/automate-studio/application/README.md (hunk 1 above)
Comment (lorisock) [reply src/docs/automate-studio/application/README.md:50]: Shoot. Nope. That was an accident.

Comment (lorisock) [reply src/docs/automate-studio/application/README.md:50]: I just checked out this branch and this text does not appear to have been deleted. Now I'm very confused.

**Pull request closed without merging**